### PR TITLE
add README with links to docs and information on how to run tests

### PR DESCRIPTION
**Pull request by paulperegud**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,47 @@
...
+
+* Added protection against chain re-orgs (https://github.com/omisego/plasma-mvp/pull/51).  
+* Added collected fee exiting for PoA operator.  
+* Added ERC20 handling.
+* Merkle tree used is of variable depth.
+* Transaction fee is implicit, not explicit.
+
+
+### Plasma MVP, confirmations, and MoreVP
+While this implementation contains confirmations, this is a temporary state as we are going to replace confirmations with [MoreVP](https://ethresear.ch/t/more-viable-plasma/2160) (Fichter, Jones) in the future. Reasons include:
```
Comment (whoisjeremylam) [inline README.md]: > While this implementation contains confirmations, this is a temporary state as we are going to replace confirmations with [MoreVP]

Would it be better to say: While this implementation contains confirmations, this is a temporary state as we are going to replace confirmations with the exit game defined in [MoreVP]

File: README.md [hunk 2]
```diff
@@ -1,3 +1,47 @@
 # Plasma Contracts
 
-Root chain contracts for Plasma MVP.
+Root chain contracts for Plasma MVP, work in progress.
+
+## Contents
+This version of the contract implements [Plasma MVP](https://ethresear.ch/t/minimal-viable-plasma/426) (Buterin, Poon, Knott). This implementation PoA scheme with one operator and multiple watchers (users). Detailed description of our child chain design is in [Tesuji document](https://github.com/omisego/elixir-omg/blob/develop/docs/tesuji_blockchain_design.md).
```
Comment (whoisjeremylam) [inline README.md]: > This implementation PoA scheme with one operator and multiple watchers (users)

This implementation is a PoA scheme with one operator and multiple watchers (users)

File: README.md [hunk 3]
```diff
@@ -1,3 +1,47 @@
...
+* Added collected fee exiting for PoA operator.  
+* Added ERC20 handling.
+* Merkle tree used is of variable depth.
+* Transaction fee is implicit, not explicit.
+
+
+### Plasma MVP, confirmations, and MoreVP
+While this implementation contains confirmations, this is a temporary state as we are going to replace confirmations with [MoreVP](https://ethresear.ch/t/more-viable-plasma/2160) (Fichter, Jones) in the future. Reasons include:
+
+* bad UX, need to propagate confirm sigs somehow;
```
Comment (whoisjeremylam) [inline README.md]: Capitalize each bullet point to stay consistent with bullet point style from above

File: README.md (hunk 1 above)
Comment (paulperegud) [reply README.md]: :+1:

File: README.md (hunk 2 above)
Comment (paulperegud) [reply README.md]: :+1:

File: README.md (hunk 3 above)
Comment (paulperegud) [reply README.md]: :+1:

**Pull request merged**